test/integration/protocol_integration_test.cc

```diff
@@ -3264,7 +3268,8 @@ TEST_P(ProtocolIntegrationTest, HeaderOnlyBytesCountUpstream) {
                "%UPSTREAM_HEADER_BYTES_SENT% %UPSTREAM_HEADER_BYTES_RECEIVED%");
   testRouterRequestAndResponseWithBody(0, 0, false);
   expectUpstreamBytesSentAndReceived(BytesCountExpectation(251, 38, 219, 18),
-                                     BytesCountExpectation(168, 13, 168, 13));
+                                     BytesCountExpectation(168, 13, 168, 13),
+                                     BytesCountExpectation(153, 5, 155, 3));
```

Comment:
3 header bytes in h3 v.s. 13 header bytes in h2 ?! If these number makes sense, can I praise QPack compression ratio?